data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+	on offer
+		conversation
+			`Coming in for a landing, something from inside your ship produces a high pitched whistle sound. It gets progressively stronger as you get closer and closer to the ringworld, until it completely shuts down, only to click back to normality and land as usual at the last moment. Looking outside, you spot several Quarg standing still, staring at your vessel, until they hastily leave, without contacting you at all.`
+				decline
+
+
+
+mission "Heliarch Recon KE"
+	invisible
+	landing
+	source "Kuwaru Efreti"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 1: active"
+	on offer
+		conversation
+			`When you approach the ringworld for landing, a couple lights start flickering on your control panel, and you can make out some whistling coming from somewhere inside your ship. It continues for a few seconds before stopping.`
+				decline
+
+
+
+mission "Heliarch Recon HU"
+	invisible
+	landing
+	source "Humanika"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 1: active"
+	on offer
+		conversation
+			`The only reason the Heliarchs must have asked you to land on Humanika is to see how the human inhabitants interact with the Quarg-controlled robots here. You land as asked, and can't seem to notice anything different with your ship.`
+				decline
+
+
+
+mission "Heliarch Recon F"
+	invisible
+	landing
+	source "Forpelog"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 1: active"
+	on offer
+		conversation
+			`	While you told them this was nothing more than a barren mining outpost, the Heliarchs still insisted on gathering intel on this small moon. Regardless of it's usefulness, you land as they asked, and let your scanners get a proper reading on this world.`
+				decline
+
+
+
+mission "Heliarch Recon G"
+	invisible
+	landing
+	source "Grakhord"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 1: active"
+	on offer
+		conversation
+			`As instructed, you land on Grakhord and let your ship work normally while the Heliarch sensor does its thing. Though they are very advanced, you doubt whether the modified scanner could penetrate the layers of rock to fully reveal the Quarg mining operation.`
+				decline
+
+
+
+event "heliarch recon break"
+
+
+
+mission "Heliarch Recon 2-A"
+	name "Scanning Preparations"
+	description "Install an Outfit Scanner, then land on <destination>, where the Heliarch will modify it to be able to scan Quarg ships the way they need it."
+	landing
+	source
+		government "Heliarch"
+		not planet "Ring of Wisdom"
+	destination "Ring of Wisdom"
+	to offer
+		has "event: heliarch recon break"
+		not "Resistance: Questions: offered"
+	on offer
+		conversation
+			`Some Heliarchs approach you as you land. You think you can recognize the Saryd of the group as one of those who asked you to land on all the Quarg-owned worlds some time ago.`
+			`	"The honorable Captain, we once again are pleased to greet," he says.`
+			`	"Another fortuitous meeting, we hope this will be."`
```

Comment:
```suggestion
			`	"Another worthwhile meeting, we hope this will be."`
```
[I do not think it means what you think it means.](https://www.merriam-webster.com/dictionary/fortuitous) - [Inigo Montoya, _The Princess Bride_](https://www.youtube.com/watch?v=G2y8Sx4B2Sk)